assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -699,6 +702,15 @@ export default Component.extend({
     }
   },
 
+  handleBulkDeleteMessage(data) {
```

Comment:
Bikeshed: deleting messages in bulk sounds like a pretty useful action on its own.

Reply:
Yes I brought that up yesterday internally about the "selected messages" actions, maybe we could have a bulk delete at some point